src/@chakra-ui/gatsby-plugin/semanticTokens.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  lightTheme as oldLightTheme,
+  darkTheme as oldDarkTheme,
+} from "../../theme"
+
+const oldLightThemeColors = oldLightTheme.colors
+const oldDarkThemeColors = oldDarkTheme.colors
+
+// define each of the old colors as a `semanticToken`:
+// `name: { _light: lightColor, _dark: darkColor }`
+const oldColors = Object.keys(oldLightThemeColors).reduce((colors, color) => {
+  const lightColor = oldLightThemeColors[color]
+  const darkColor = oldDarkThemeColors[color]
+
+  if (typeof lightColor !== "string" || typeof darkColor !== "string") {
+    return colors
+  }
+
+  return {
+    ...colors,
+    [color]: { _light: lightColor, _dark: darkColor },
+  }
+}, {})
+
+const semanticTokens = {
+  colors: {
+    // define old colors from the old theme as semanticTokens to transition
+    // from emotion components to chakra
+    // TODO: remove these colors as we migrate away from them
+    ...oldColors,
```

Comment:
This is to do the migration as seamless as possible. Detailed in the migration guide under the `## Theme colors` header. https://github.com/ethereum/ethereum-org-website/pull/7368
The idea is that we keep using the same color variable names.